tests/unit/test_logging.py

```diff
@@ -257,3 +259,48 @@ def test_bootstrap_logger(caplog: _pytest.logging.LogCaptureFixture) -> None:
     )
 def test_indent(key, value, color, level, labels, expected):
     assert indent(key, value=value, color=color, level=level, labels=labels) == expected
+
+
+@pytest.mark.parametrize(
+    ('logger_topics', 'message_topic', 'filter_outcome'),
+    [
+        # (
+        #     logger topics,
+        #     message topic,
+        #   expected outcome of `QietnessFilter.filter()`
+        # )
+        (
+            set(),
+            None,
+            True
+            ),
+        (
+            set(),
+            Topic.KEY_NORMALIZATION,
+            False
+            ),
+        (
+            {Topic.KEY_NORMALIZATION},
+            Topic.KEY_NORMALIZATION,
+            True
+            ),
+        ],
```

Comment:
Could you add 'logger topic, message no topic' and 'logger topic, message different topic' cases?

Reply:
Added the first one in bbea982d, and I can't add the second because there's only one topic so far, therefore I can't construct two different topics, one for the logger, and one for the message. I plan to add more topics though, so added as a TODO comment.